runtime/src/lib.rs

```diff
@@ -364,7 +364,7 @@ impl orml_tokens::Config for Runtime {
 	type ExistentialDeposits = AssetRegistry;
 	type OnDust = Duster;
 	type MaxLocks = MaxLocks;
-	type DustRemovalWhitelist = Nothing;
+	type DustRemovalWhitelist = Duster;
```

Comment:
kinda random that `Duster.contains` actually a set on non-dustable accounts, but I guess is acceptable

Reply:
i would rather move it to some sub trait like `Duster.dustRemovalWhitelist` that would implement the contains if it's doable

Reply:
good point - I have not looked at it that way.

It is doable. Will do.